.circleci/config.yml

```diff
@@ -220,6 +212,7 @@ jobs:
 
   check:
     executor: golang
+    resource_class: large
```

Comment:
The real reason for this PR as far as I understand is this line because golanglint-ci has not enough memory.
We need to add this reason either in PR or here as a little comment (I prefer the latter)